pandas/tests/io/sas/test_sas7bdat.py

```diff
@@ -183,6 +183,22 @@ def test_date_time(datapath):
     tm.assert_frame_equal(df, df0)
 
 
+def test_many_columns(datapath):
+    fname = datapath("io", "sas", "data", "many_columns.sas7bdat")
+    df = pd.read_sas(fname, encoding='latin-1')
+    fname = datapath("io", "sas", "data", "many_columns.csv")
+    df0 = pd.read_csv(fname, encoding='latin-1',
+                      dtype={
+                          'DATASRC': np.dtype('S'),
+                          'PDDOCID': np.dtype('S'),
+                          'VISIT_NO': np.dtype('S'),
+                          'ecgrtxt': np.dtype('S'),
+                          'ecgrtime': np.dtype('d'),
+                          'nvitl1': np.dtype('d')
+                      })
```

Comment:
Generally, we would prefer to construct the expected DataFrame from scratch.  This is one of the motivations for having a small, reproducible example.

Reply:
Hi @gfyoung, I'd generally agree, but in this case, I don't have a good way of constructing another test example. I could perhaps use a more type-safe format than csv, as the reference file, do you have any good ideas?

Reply:
Gotcha.  Let's see what you have here: one header and three rows IIUC.  Do you need that many columns to construct a breaking example?

Reply:
Yeah. The problem occurs as a consequence of the number of columns. Otherwise SAS will save all column info on one page, and the problem will not be replicated.

Reply:
Hmm...I see.  Alright, let's leave this as is for now then. 👍